xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/displayer_page.vm

```diff
@@ -1 +1,40 @@
-#template('displayer_dblist.vm')
+#if ($type == 'edit' || $type == 'search')
+  #if ($field.getProperty('displayType').value == 'input' && $field.getProperty('picker').value == 1)
+    #xpropertySuggestInputDisplayer($field $prefix $name $value)
+  #else
+    $doc.displayEdit($field, $prefix, $object)
+  #end
+#elseif ($type == 'view')
+  #set ($pages = [])
+  #foreach($pageReference in $value)
+    #set ($discard = $pages.add("#displayPageReference($pageReference)"))
+  #end
+  #if ($pages.size() > 0)
+    $stringtool.join($pages, '-')
+  #else
+    #displayPageReference($value)
+  #end
+#elseif ($type == 'rendered')
+  $doc.displayRendered($field, $prefix, $object)
+#elseif ($type == 'hidden')
+  $doc.displayHidden($field, $prefix, $object)
+#else
+  ## In order for the custom displayer to be taken into account, the result of its evaluation with an unknown display
+  ## mode must not be empty. Let's output something.
+  Unknown display mode.
+#end
+
+#macro(displayPageReference $pageReference)
+  #set ($pageDoc = $xwiki.getDocument($pageReference))
+  #if ("$!pageDoc" == "")
+    $pageReference
+  #else
+    #if ($pageDoc.isNew())
+      <span class="wikicreatelink">
+        <a href="$pageDoc.getURL('create')">$pageDoc.getRenderedTitle('plain/1.0')</a>
```

Comment:
``$pageDoc.plainTitle`` is shorter. In any case, you need to ``$escapetool.xml(...)`` as the plain title can have XML / HTML special characters.